crates/uv/tests/run.rs

```diff
@@ -197,6 +197,81 @@ fn run_args() -> Result<()> {
     Ok(())
 }
 
+/// Run without specifying any argunments.
+/// This should list the available scripts.
+#[test]
+fn run_no_args() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let pyproject_toml = context.temp_dir.child("pyproject.toml");
+    pyproject_toml.write_str(indoc! { r#"
+        [project]
+        name = "foo"
+        version = "1.0.0"
+        requires-python = ">=3.8"
+        dependencies = []
+
+        [build-system]
+        requires = ["setuptools>=42"]
+        build-backend = "setuptools.build_meta"
+        "#
+    })?;
+
+    // Run without specifying any argunments.
+    #[cfg(not(windows))]
+    uv_snapshot!(context.filters(), context.run(), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+    Provide a command or script to invoke with `uv run <command>` or `uv run script.py`.
+
+    The following scripts are available:
+
+    python
+    python3
+    python3.12
+
+    See `uv run --help` for more information.
+
+    ----- stderr -----
+    Resolved 1 package in [TIME]
+    Prepared 1 package in [TIME]
+    Installed 1 package in [TIME]
+     + foo==1.0.0 (from file://[TEMP_DIR]/)
+    "###);
+
+    #[cfg(windows)]
+    uv_snapshot!(context.filters(), context.run(), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+    Provide a command or script to invoke with `uv run <command>` or `uv run script.py`.
+
+    The following scripts are available:
+
+    activate.bat
+    activate.csh
+    activate.fish
+    activate.nu
+    activate.ps1
+    activate_this.py
+    deactivate.bat
+    pydoc.bat
+    python.exe
+    pythonw.exe
```

Comment:
We'll probably also want to exclude the `.exe` suffix since it's implied during invocation.

Reply:
👍 I didn't know that. I need a Windows setup to test it. I will check the codebase or take your word for it 😄